Application/src/view/visuals/Draw.java

```diff
@@ -291,13 +290,45 @@ private void renderClass() {
      * Places the classes in the class diagram
      */
 
-    void renderClassDiagramClass(){
+    void renderClassDiagramClass() {
         if (allClassDiagramClasses.size() == 0) return;
-        int space = ((int)this.canvas_class.getWidth())/this.allClassDiagramClasses.size();
-        for(int i = 0; i < allClassDiagramClasses.size(); i++) {
-            int x = (space/2) + (i * space);
-            int y = 50 + (space/2)/4;
-            allClassDiagramClasses.get(i).place(new Coordinates(x,y), (space/2));
+        int space = ((int) this.canvas_class.getWidth()) / this.allClassDiagramClasses.size();
```

Comment:
Use the column size to determine the space e.g. 
int space = ((int) this.canvas_class.getWidth()) / (this.allClassDiagramClasses.size()/col);